pkl/builtins/dprint.pkl

```diff
@@ -0,0 +1,8 @@
+import "../Config.pkl"
+
+dprint = new Config.Step {
+    glob = "**/*"
+    check = "dprint check --allow-no-files {{ files }}"
```

Comment:
This actually outputs a diff, but not the conventional kind, so not applicable for check_diff I think.